jobserv/worker.py

```diff
@@ -223,8 +223,11 @@ def run_monitor_workers():
     log.info("worker monitor has started")
     try:
         while True:
-            log.debug("checking workers")
-            _check_workers()
+            if os.path.isdir(WORKER_DIR):
```

Comment:
So if we do not have the work dir that is fine.
Wil we use the current dir?

Should we not also check/test out this scenario in https://github.com/foundriesio/jobserv/blob/1af5ae7d993b832ee796242e26096e9481956650/tests/test_worker.py#L26?

Reply:
What is surge?

Reply:
JobServ has a concept called "surge workers". In the old days you can enlist your home server as a surge worker. This allows them to participate in CI when there's a heavy load. We don't do this for any actual customers.